backend/backend.go

```diff
@@ -20,19 +20,67 @@ const (
 	TableCENReport = "CENReport"
 
 	// Default Conn String 
-	DefaultConnString = "[change this to your server]"
+	DefaultConnString = "root:2Nemesis234@/conn"
 )
 
 // Backend holds a client to connect  to the BigTable backend
 type Backend struct {
 	db *sql.DB
 }
 
+//
+// Brian Dickson's added comments for understanding the SQL elements used/required
+//
+// OLD DEFINITIONS, WRONG SIZE FOR cenKey
+//
+//  CREATE TABLE `CENKeys` (
+//     `cenKey`   varchar(32) DEFAULT "", 
+//     `reportID` varchar(64) DEFAULT "",
+//     `reportTS` int,
+//     PRIMARY KEY (`cenKey`, `reportID`),
+//     KEY (`reportID`),
+//     KEY (`reportTS`),
+//     KEY (`cenKey`)
+//  );
+//  
+//  CREATE TABLE `CENReport` (
+//     `reportID` varchar(64) DEFAULT "",
+//     `report`     varchar(4000) DEFAULT "",
+//     `reportMimeType` varchar(64) DEFAULT "",
+//     `reportTS` int,
+//     PRIMARY KEY (`reportID`),
+//     KEY (`reportTS`)
+//  );
+//
+//
+// NEW DEFINITIONS, RIGHT SIZE FOR cenKey
+//
+//  CREATE TABLE `CENKeys` (
+//     `cenKey`   varchar(64) DEFAULT "", 
+//     `reportID` varchar(64) DEFAULT "",
+//     `reportTS` int,
+//     PRIMARY KEY (`cenKey`, `reportID`),
+//     KEY (`reportID`),
+//     KEY (`reportTS`),
+//     KEY (`cenKey`)
+//  );
+//  
+//  CREATE TABLE `CENReport` (
+//     `reportID` varchar(64) DEFAULT "",
+//     `report`     varchar(4000) DEFAULT "",
+//     `reportMimeType` varchar(64) DEFAULT "",
+//     `reportTS` int,
+//     PRIMARY KEY (`reportID`),
+//     KEY (`reportTS`)
+//  );
+//
+//
+
 // CENReport payload is sent by client to /cenreport when user reports symptoms
 type CENReport struct {
 	ReportID        string `json:"reportID,omitempty"`
 	Report          []byte `json:"report,omitempty"`  // this is expected to be a JSON blob but the server doesn't need to parse it
-	CENKeys         string `json:"cenKeys,omitempty"` // comma separated list of base64 AES Keys
+	CENKeys         string `json:"cenKeys,omitempty"` // comma separated list of hex AES-128 Keys (COMMENT WAS: base64 AES Keys)
```

Comment:
Why do we need to retain the old comment?

Reply:
Good point. Old habits die hard, this is a practice from before the days of having comments in the review process itself. Feel free to kill it.